lib/connections/quic_dial.go

```diff
@@ -4,7 +4,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// +build go1.12,!noquic
+// +build go1.12,!noquic,!go1.16
```

Comment:
While at it: Quic/qtls apparently removed support for go1.13 a while ago, so the actual lower bound is now go1.14:

https://github.com/marten-seemann/qtls/commit/6befe96d4cf783737540378ecef36e171cdca49c